test/harry/main/org/apache/cassandra/harry/dsl/SingleOperationBuilder.java

```diff
@@ -36,8 +35,8 @@ public interface SingleOperationBuilder
      * Insert _specific values_ into _specific_ row. Rows are ordered by clustering key and
      * numbered from 0 to maxRows
      */
-    SingleOperationBuilder insert(int rowIdx, long[] vds);
-    SingleOperationBuilder insert(int rowIdx, long[] vds, long[] sds);
+    SingleOperationBuilder insert(int rowIdx, int[] vds);
+    SingleOperationBuilder insert(int rowIdx, int[] vds, int[] sds);
```

Comment:
Should we be clear here in term of naming that these are indexes and no longer actual ads? `SingleOperationVisitBuilder` is a little clearer, of course...

Reply:
Well, the inserts happen and the SUT matches, but `validateAll()` doesn't like it :)

Reply:
I see the right `ValueHelper#overrides` bijection getting the mapping, but then nothing accesses the overrides?

Reply:
Good point, made them

```
    SingleOperationBuilder insert(int rowIdx, long[] valueIdxs);
    SingleOperationBuilder insert(int rowIdx, long[] valueIdxs, long[] sValueIdxs);
```

Reply:
> Well, the inserts happen and the SUT matches, but validateAll() doesn't like it :)

Could you elaborate a bit? `history.valueOverrides().override(column.name, i, override);` is just used as a convenience method to quickly access override generator.

But we do a swap of `Schema` in `HistoryBuilder` to make sure all child generators are getting the correct (overriden) generators:

```
        this.schema = schema.withCkGenerator(this.ckGenerator, this.ckGenerator.columns)
                            .withColumns(valueHelper.regularColumns, valueHelper.staticColumns);
```